reqrespupdate/workflow.go

```diff
@@ -0,0 +1,119 @@
+package reqrespupdate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	UpdateHandler = "update"
+)
+
+var (
+	ErrBackoff = errors.New("trying to continue as new")
+)
+
+// UppercaseWorkflow is a workflow that accepts requests to uppercase strings
+// via updates and provides a response.
+func UppercaseWorkflow(ctx workflow.Context, rejectUpdateOnPendingContinueAsNew bool) error {
```

Comment:
Would consider `UppercaseWorkflowOptions` struct here just so that what the bool value means caller-side is clearer when set as a field (not required, no biggie)